phpunit

```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-23 14:50 UTC
+// Cache-Id-Version: 2016-03-25 09:44 UTC
```

Comment:
you only need to change it here when you change the file (i.e. when reverting the requirement on 3.1.1. mocks)